pkg/baseline/packageIndex.json

```diff
@@ -9,8 +9,9 @@
       "BaselineVersion": "4.4.0",
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
-        "4.1.1.0": "4.4.0",
-        "4.1.2.0": "4.4.0"
+        "4.1.1.0": "4.5.0",
```

Comment:
I know @StephenBonikowsky made this change to make validation happy but I just have a hard time to understand the magic here :) - both 4.1.1.0 and 4.1.2.0 were mapping to package version 4.4.0 but now 4.1.1.0 has to be changed to be mapped to 4.5.0 but not for 4.1.2.0. @ericstj is this expected?